spectroscopy/code_src/herschel_functions.py

```diff
@@ -55,84 +57,89 @@ def Herschel_get_spec(sample_table, search_radius_arcsec, datadir,
         separate data directory (for example "[datadir]/HST/" for HST data).
     delete_downloaded_data: `bool`, optional
         Should the tarfiles be deteled after spectra are extracted?
-        
+
     Returns
     -------
     df_spec : MultiIndexDFObject
         The main data structure to store all spectra
-        
+
     '''
 
-    ## Initialize multi-index object:
+    # Initialize multi-index object:
     df_spec = MultiIndexDFObject()
 
     for stab in sample_table:
         search_coords = stab["coord"]
         print("working on object", stab["label"])
-        
-        #first find the object ids from herschel then download the data for each observation id
-        #query_hsa_tap doesn't accept an upload_table, so do this so do this as a for loop over each instrument and object..
- 
+
+        # first find the object ids from herschel then download the data for each observation id
+        # query_hsa_tap doesn't accept an upload_table, so do this so do this as a for loop over each instrument and object..
+
         for instrument_name in ['PACS', 'SPIRE']:
-            querystring = "select observation_id from hsa.v_active_observation join hsa.instrument using (instrument_oid) where contains(point('ICRS', hsa.v_active_observation.ra, hsa.v_active_observation.dec), circle('ICRS', "+str(search_coords.ra.deg)+", " + str(search_coords.dec.deg) +", " + str(search_radius_arcsec) +"))=1 and hsa.instrument.instrument_name='"+str(instrument_name)+"'"
+            querystring = "select observation_id from hsa.v_active_observation join hsa.instrument using (instrument_oid) where contains(point('ICRS', hsa.v_active_observation.ra, hsa.v_active_observation.dec), circle('ICRS', "+str(
+                search_coords.ra.deg)+", " + str(search_coords.dec.deg) + ", " + str(search_radius_arcsec) + "))=1 and hsa.instrument.instrument_name='"+str(instrument_name)+"'"
```

Comment:
Here's one example. Line 79 is still 232 characters long.

Reply:
this is what you got with the auto formatted? I agree that it's still very ugly and not really an improvement compared to the one super long line.

Reply:
Yes, just autopep8 so far, no manual adjustments.